qiime2/sdk/tests/test_config.py

```diff
@@ -215,6 +218,10 @@ def test_no_vendored_fp(self):
             self.assertEqual(list_execution_contexts, self.tpool_expected)
             self.assertEqual(dict_execution_contexts, self.tpool_expected)
 
+    @pytest.mark.skipif(platform.system() == 'Darwin' and
+                        datetime.today().strftime('%m-%d') < '12-13',
```

Comment:
This will technically go back to passing in Jan

Reply:
I figured we would have killed this by then, but I can change it if you would like